osu.Android/OsuGameActivity.cs

```diff
@@ -23,8 +30,46 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             base.OnCreate(savedInstanceState);
 
+            OnNewIntent(Intent);
```

Comment:
this seems a bit suspicious. would want to test if it's required, and if so, leave an inline comment explaining why.